src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -34,15 +34,21 @@ type State = {
   incidentList: Incident[];
 };
 
+function getQueryStatus(status: any) {
+  return status === undefined || status === '' ? DEFAULT_QUERY_STATUS : status;
```

Comment:
`status` _could_ be a list if it was included multiple times in the query string. But that might not be a scenario worth worrying about.